iot_packet_verifier/tests/protobufs.rs

```diff
@@ -0,0 +1,120 @@
+/// Loading & processing of encoded Protobufs and bypass AWS S3 during ingest.
+extern crate iot_packet_verifier as pv;
+
+use helium_proto::services::router::PacketRouterPacketReportV1;
+use helium_proto::Message;
+use helium_proto::{GatewayBookkeeping, IotPacketVerifierBookkeeping, OuiBookkeeping};
+
+// https://developers.google.com/protocol-buffers/docs/encoding#structure
+static ENCODED_PROTOBUF_BYTES: [[u8; 66]; 6] = [
```

Comment:
Is this also included in a non test build?

Reply:
Fixed in commit e0e1c6a.

Thanks for the reminder.  Still getting my Rust sea legs after primarily touching Erlang...